designer/server/src/test/scala/pl/touk/nussknacker/test/processes/WithScenarioActivitySpecAsserts.scala

```diff
@@ -138,3 +203,12 @@ trait WithScenarioActivitySpecAsserts
   }
 
 }
+
+object WithScenarioActivitySpecAsserts extends WithBusinessCaseRestAssuredUsersExtensions {
+
+  final case class ScenarioActivitiesResponseWrapper(validatableResponse: ValidatableResponse) {
+    def extractString(activityIndex: Int, path: String): String =
```

Comment:
it's ok, but IMO it should be more specialized. Like `getAddedAttachementId: String`.

It's because the functionality is not so "symmetric" currently. We verify that the attachment was added to the scenario without other activities (see the body matcher). And then we allow getting any activity, but there are only two of them :P

So, let's make it more "symmetric"